__tests__/intergration.test.js

```diff
@@ -148,6 +148,31 @@ describe('/api/articles', () => {
             expect(response.body.msg).toBe('Not Found')
         })
     });
+    it('should respond with all articles based on topic given in query ', () => {
+        return request(app).get('/api/articles').query({ topic: 'mitch' })
+        .expect(200)
+        .then((response) => {
+            response.body.articles.forEach((article) => {
+                expect(article).toHaveProperty('author');
+                expect(article).toHaveProperty('title');
+                expect(article).toHaveProperty('article_id');
+                expect(article).toHaveProperty('topic');
+                expect(article).toHaveProperty('created_at');
+                expect(article).toHaveProperty('votes');
+                expect(article).toHaveProperty('article_img_url');
+                expect(article).toHaveProperty('body');
```

Comment:
The _body_ property isn't expected on this ticket, that's only expected when fetching an individual article. It does no harm to fetch it, but depends how strictly you want to follow the ticket